examples/movielense_multihot_example.ipynb

```diff
@@ -0,0 +1,1309 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "movies = cudf.from_pandas(movies)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We load the movie ratings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>userId</th>\n",
+       "      <th>movieId</th>\n",
+       "      <th>rating</th>\n",
+       "      <th>timestamp</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>1</td>\n",
+       "      <td>296</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>1147880044</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>1</td>\n",
+       "      <td>306</td>\n",
+       "      <td>3.5</td>\n",
+       "      <td>1147868817</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>1</td>\n",
+       "      <td>307</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>1147868828</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>1</td>\n",
+       "      <td>665</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>1147878820</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>1</td>\n",
+       "      <td>899</td>\n",
+       "      <td>3.5</td>\n",
+       "      <td>1147868510</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   userId  movieId  rating   timestamp\n",
+       "0       1      296     5.0  1147880044\n",
+       "1       1      306     3.5  1147868817\n",
+       "2       1      307     5.0  1147868828\n",
+       "3       1      665     5.0  1147878820\n",
+       "4       1      899     3.5  1147868510"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ratings = cudf.read_csv('/raid/data/ml/ml-25m/ratings.csv')\n",
```

Comment:
We should use the directory variable (BASE_DIR) instead of hardcoding the path:
```suggestion
    "ratings = cudf.read_csv(os.path.join(BASE_DIR, "ml-25m", ratings.csv'))\n",
```